csrc/cutlass_extensions/epilogue/scaled_mm_epilogues_c3x.hpp

```diff
@@ -36,13 +36,13 @@ struct ScaledEpilogueBase {
   // Don't want to support nullptr by default
   template <typename T, bool EnableNullPtr = false>
   using ColLoad = cutlass::epilogue::fusion::Sm90ColBroadcast<
-      0 /*Stages*/, typename EpilogueDescriptor::TileShape, T,
+      0 /*Stages*/, typename EpilogueDescriptor::TileShape, T, T,
```

Comment:
Is this change required by the CUTLASS 3.6.0 upgrade?

Reply:
Yes